examples/with-okta-multi-tenant-pkce-flow/api-server/okta.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/supertokens/supertokens-golang/recipe/thirdparty/tpmodels"
+	"github.com/supertokens/supertokens-golang/supertokens"
+)
+
+type OktaConfig struct {
+	ClientID   string
+	OktaDomain string
+}
+
+var oktaProvider = tpmodels.TypeProvider{
```

Comment:
- make this a function that returns the provider type, and to that function, the user has to pass in the oktaRedirectURI; OR
- You can directly form it based on the websiteDomain/websiteBasePath/callback/okta

Reply:
done